system-addon/data/content/assets/glyph-modal-delete-32.svg

```diff
@@ -0,0 +1,10 @@
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<svg xmlns="http://www.w3.org/2000/svg" width="32" height="32" viewBox="0 0 32 32">
```

Comment:
we used to have `svgo` in package.json and that could be run to compress these things down. Not sure why it was removed though. I guess you can still npm install it and run it.